component/src/main/java/io/siddhi/extension/io/nats/sink/nats/NATSStreaming.java

```diff
@@ -0,0 +1,137 @@
+            streamingConnection = streamingConnectionFactory.createConnection();
+        } catch (IOException e) {
+            throw new ConnectionUnavailableException("Error in Siddhi App " + siddhiAppName + " while connecting to " +
+                    "NATS server endpoint " + Arrays.toString(natsUrls) + " at destination: " + destination.getValue()
+                    , e);
+        } catch (InterruptedException e) {
+            throw new ConnectionUnavailableException("Error in Siddhi App " + siddhiAppName + " while connecting to" +
+                    " NATS server endpoint " + Arrays.toString(natsUrls) + " at destination: " +
+                    destination.getValue() + ". The calling thread is interrupted before the connection " +
+                    "can be established.", e);
+        }
+        isConnected.set(true);
+    }
+
+    @Override
+    public void publishMessages(Object payload, byte[] messageBytes, DynamicOptions dynamicOptions)
+            throws ConnectionUnavailableException {
+        String subjectName = destination.getValue(dynamicOptions);
+        try {
+            if (!isConnected.get()) {
+                if (streamingConnection != null) {
+                    streamingConnection.close();
+                }
+                createNATSClient(); // this method is only use to create the client
+            }
+            streamingConnection.publish(subjectName, messageBytes,
+                    new AsyncAckHandler(siddhiAppName, natsUrls, payload, natsSink, dynamicOptions));
+        } catch (IOException e) {
+            throw new SiddhiAppRuntimeException("Error sending message to destination:" + subjectName, e);
+        } catch (InterruptedException e) {
+            throw new SiddhiAppRuntimeException("Error sending message to destination:" + subjectName
+                    + ".The calling thread is interrupted before the call completes.", e);
+        } catch (TimeoutException e) {
+            throw new SiddhiAppRuntimeException("Error sending message to destination:" + subjectName
+                    + ".Timeout occured while trying to ack.", e);
+        }
+    }
+
+    @Override
+    public void disconnect() {
+        if (streamingConnection != null) {
+            try {
+                streamingConnection.close();
+            } catch (IOException | TimeoutException | InterruptedException e) {
+                log.error("Error disconnecting the Stan receiver in Siddhi App " + siddhiAppName +
+                        " when publishing messages to NATS endpoint " + Arrays.toString(natsUrls) + " . " +
+                        e.getMessage(), e);
+            }
+        }
+    }
+
+    class NATSConnectionLostHandler implements ConnectionLostHandler {
+
+        @Override
+        public void connectionLost(StreamingConnection streamingConnection, Exception e) {
+            log.error("Exception occurred in Siddhi" + " App " + siddhiAppName + " when publishing messages " +
```

Comment:
Use single quotes to highlight siddhi app names and stream ids in logs.Check-in all places
```suggestion
            log.error("Exception occurred in Siddhi App '" + siddhiAppName + "' when publishing messages " +
```

Reply:
Fixed in https://github.com/siddhi-io/siddhi-io-nats/pull/37/commits/949cd00960288204191b056f20e5dcfe32afcb10